scan/lib/scan/detect_values.rb

```diff
@@ -138,7 +138,7 @@ def self.detect_destination
       elsif Scan.project.tvos?
         Scan.config[:destination] = Scan.devices.map { |d| "platform=tvOS Simulator,id=#{d.udid}" }
       else
-        Scan.config[:destination] = ["platform=OS X"]
+        Scan.config[:destination] = ["platform=macOS"]
```

Comment:
You can access the version using `Helper.xcode_version`, which will return the version as a string. 
Please do some testing around this, so that everything works with both the new and the old Xcode version